docs/ref/csrf.txt

```diff
@@ -222,24 +222,34 @@ The CSRF protection is based on the following things:
 1. A CSRF cookie that is set to a random value (a session independent nonce, as
```

Comment:
See my  other comment about correctness, but the tern was changed anyway to avoid confusion.